tools/identity-resolution/identity-resolution/Helpers/GitHubAadAliasConverter.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Net.Http;
+using Azure.Sdk.Tools.NotificationConfiguration.Models;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Sdk.Tools.NotificationConfiguration.Helpers
+{
+    public class GitHubToAadAliasResolver
+    {
+        /// <summary>
+        /// Create a new GitHubToMsAliasResolver
+        /// </summary>
+        /// <param name="clientId">AAD Client/App Id</param>
+        /// <param name="clientSecret">AAD Client/App Secret</param>
+        /// <param name="tenantID">AAD Tenant Id</param>
+        /// <param name="logger">Logger</param>
+        public GitHubToAadAliasResolver(
+            string clientId,
+            string clientSecret,
+            string tenantID,
+            ILogger<GitHubNameResolver> logger)
+        {
+            try
+            {
+                this.opsAuthToken = new AadTokenCredential(clientId, clientSecret, tenantID).GenerateAadToken();
+            }
+            catch (Exception ex)
+            {
+                logger.LogError("Failed to generate aad token. " + ex.Message);
+            }
+            this.logger = logger;
+        }
+
+        private readonly string opsAuthToken;
+        private readonly ILogger<GitHubNameResolver> logger;
+
+        public string GetMsAliasFromGithub(string githubUserName)
+        {
+            HttpClient client = InitializeOpenSourceClient(opsAuthToken);
```

Comment:
I would also move this into an `Initialize()` method along with the `GenerateAadToken()` call. We don't need to be creating multiple http clients. I'm not sure how many times this method gets called per-process, but `HttpClient` is intended to be shared vs. created per call.

Reply:
Yes, moved.